pkg/types/aws_resource_manager.go

```diff
@@ -45,7 +48,12 @@ type AWSResourceManager interface {
 	// higher-level reonciler determines whether or not the desired differs
 	// from the latest observed and decides whether to call the resource
 	// manager's Update method
-	Update(context.Context, /*desired*/ AWSResource, /*latest*/ AWSResource, *ackcompare.Reporter) (AWSResource, error)
+	Update(
+		context.Context,
+		AWSResource, // desired
+		AWSResource, // latest
+		*ackcompare.Reporter,
+	) (AWSResource, error)
```

Comment:
FYI, it's better to leave style-only changes like this to a separate style-only PR. Makes reviewing the code easier since it separates functional code changes from non-functional code changes.

Reply:
Agreed 👍 gofmt was automatically doing weird comment style change... i'll make this change in a different PR